.gitignore

```diff
@@ -35,3 +35,6 @@ profile.json
 
 # macOS
 **/.DS_Store
+
+# IntelliJ RustRover
+.idea/
```

Comment:
Generally these things should go in your user `.gitignore` file instead, since they're not specific to the project https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file

Reply:
Yes, I'm aware of user-specific `.gitignore`. 
I just thought if `.gitignore` contains some mac specific files it's also valid to add `IntelliJ` files too:
```
# macOS
**/.DS_Store
```

Reply:
Yes, arguably it shouldn't contain those either, but I am not the only one managing the contents of the file :D